test/Microsoft.ML.Tests/Scenarios/Api/Visibility.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.ML.Runtime.Data;
+using Xunit;
+
+namespace Microsoft.ML.Tests.Scenarios.Api
+{
+    public partial class ApiScenariosTests
+    {
+        /// <summary>
+        /// Visibility: It should, possibly through the debugger, be not such a pain to actually
+        /// see what is happening to your data when you apply this or that transform. E.g.: if I
+        /// were to have the text "Help I'm a bug!" I should be able to see the steps where it is
+        /// normalized to "help i'm a bug" then tokenized into ["help", "i'm", "a", "bug"] then
+        /// mapped into term numbers [203, 25, 3, 511] then projected into the sparse
+        /// float vector {3:1, 25:1, 203:1, 511:1}, etc. etc.
+        /// </summary>
+        [Fact]
+        void Visibility()
+        {
+            var dataPath = GetDataPath(SentimentDataPath);
+            var testDataPath = GetDataPath(SentimentTestPath);
+
+            using (var env = new TlcEnvironment(seed: 1, conc: 1))
+            {
+                // Pipeline.
+                var loader = new TextLoader(env, MakeSentimentTextLoaderArgs(), new MultiFileSource(dataPath));
+
+                var trans = TextTransform.Create(env, MakeSentimentTextTransformArgs(false), loader);
+                
+                // In order to find out available column names, you can go through schema and check
+                // column names and appropriate type for getter.
+                for( int i=0; i< trans.Schema.ColumnCount; i++)
+                {
+                    var columnName = trans.Schema.GetColumnName(i);
+                    var columnType = trans.Schema.GetColumnType(i).RawType;
+                }
```

Comment:
Little weird to just get column names and not to do anything with them, right? may be reuse them somehow at line 39-41? Make it look as if you are looking for a column a given column name, then store it and document why you expect this column to be there in the first place, like Features_TransformedText is produced as intermediate column by the text transform. Such examples and documentation will be very useful for people who are getting started with ML.NET.